src/plugin/timezone/index.js

```diff
@@ -9,6 +9,15 @@ const typeToPos = {
   second: 5
 }
 
+const dateTimeFormatDefaults = {
+  year: 'numeric',
+  month: '2-digit',
+  day: '2-digit',
+  hour: '2-digit',
+  minute: '2-digit',
+  second: '2-digit'
+}
+
 // Cache time-zone lookups from Intl.DateTimeFormat,
 // as it is a *very* slow method.
 const dtfCache = {}
```

Comment:
I think we can use a Map here in stead of `Object`. That's maybe a little bit faster. What do you think?